questions/includes/fedora/coding.yml

```diff
@@ -2,30 +2,38 @@ tree:
   children:
     - title: Python
       subtitle: a dynamic language for rapid development
-      segue1: So, you enjoy traceback-driven development?
+      segue1: So, you enjoy traceback-driven development?  What could go wrong?
       segue2: Are you interested in hacking on...
       children: coding/python.yml
 
     - title: C
       subtitle: close to the metal
-      href: https://lkml.org
+      segue1: So, you think OOP is for hipsters?  Let me tell you, back in my day...
+      segue2: How about working on...
+      children: coding/c.yml
 
     - title: Haskell
       subtitle: a nondynamic language for rapid development
-      segue1: Yay, types!
+      segue1: So, programming without side-effects is what you're all about?  Sounds useful!
```

Comment:
Wait.  Is this too harsh?  I want them to be punchy but I don't want to be rude :(